src/core/views.py

```diff
@@ -3002,8 +3005,21 @@ def view_proposal_review_decision(
                 )
             )
 
-    if request.POST:
+    if request.POST and 'accept' in request.POST or 'decline' in request.POST:
```

Comment:
Be very careful here. From what I can tell this is basically
`(if request.POST and 'accept' in request.POST) or 'decline' in request.POST:`

i.e. if `request.POST` is `None` this will crash

I would recommend you use parentheses (also makes the intention clear), so
`if request.POST and 'accept' in (request.POST or 'decline' in request.POST):`

Reply:
(edited the above comment)